### PR TITLE
better names

### PR DESCRIPTION
**Pull request by ashwin2802**
solves #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,19 +7,19 @@ For more information on the syntax and usage of rosinstall, please refer to [thi
     This is the sample rosinstall file for cloning the repository [catkin_simple](https://github.com/catkin/catkin_simple) using https. More repositories can be added as required.  
     * template_ssh.rosinstall
     This is the sample rosinstall file for cloning the same repository as above, but using ssh keys.  
-* lib_package_template  
+* pcl_ransac  
 This folder contains the code for the custom libraries that you have written for use in your ROS package. this should not contain any ROS code, and should be modular so that it can be used in other packages as well.  
     * include  
-        * lib_package_template  
+        * pcl_ransac  
             * template.hpp
```
Comment (gajena) [inline README.md]: check all the file name in readme.

File: pcl_ransac/CMakeLists.txt [hunk 2]
```diff
@@ -15,13 +15,13 @@ add_definitions(${PCL_DEFINTIONS})
 find_package(Eigen3 REQUIRED)
 include_directories(${Eigen3_INCLUDE_DIRS})
 
-cs_add_library(${PROJECT_NAME}_lib src/template.cpp)
-target_link_libraries(${PROJECT_NAME}_lib ${PCL_LIBRARIES})
+cs_add_library(${PROJECT_NAME}_lib src/pcl_ransac.cpp)
+target_link_libraries(${PROJECT_NAME}_lib ${PCL_LIBRARIES} ${Eigen3_LIBRARIES})
```
Comment (gajena) [inline pcl_ransac/CMakeLists.txt]: why are you adding Eigen now?

File: pcl_ransac_ros/launch/pcl_ransac.launch [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<launch>
+
+    <node pkg="pcl_ransac_ros" type="pcl_ransac_ros_node" name="pcl_ransac_ros_node" output="screen">
+        <rosparam command="load" file="$(find pcl_ransac_ros)/cfg/PCGenParams.yaml" />
```
Comment (gajena) [inline pcl_ransac_ros/launch/pcl_ransac.launch]: change param file name.

File: pcl_ransac_ros/include/pcl_ransac_ros/CloudModelFitter.hpp [hunk 4]
```diff
@@ -0,0 +1,36 @@
+#include <ros/ros.h>
```
Comment (gajena) [inline pcl_ransac_ros/include/pcl_ransac_ros/CloudModelFitter.hpp]: change file name

**Review by gajena — CHANGES_REQUESTED** [review]

File: pcl_ransac/CMakeLists.txt (hunk 2 above)
Comment (ashwin2802) [reply pcl_ransac/CMakeLists.txt]: removed since unnecessary. include_directories gives us the required definitions already/

File: README.md [hunk 5]
```diff
@@ -46,41 +46,26 @@ This package contains all the custom [actions](http://wiki.ros.org/actionlib/Tut
 * pcl_ransac_ros  
 This contains the main ROS package. All your package-specific code goes here.  
     * cfg  
-        * template.cfg
```
Comment (Kshitij1K) [inline README.md:49]: Maybe let this part remain? It has references for writing dynamic reconfigure files

**Review by gajena — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (ashwin2802) [reply README.md:49]: we can add that later, when we add dyncfg to the template

**Comment by Kshitij1K** [comment]
Cool, just a suggestion we should probably have empty files in other directories as well (and their corresponding sections in the README), just so that people know that they are supposed to exist and their purpose.

**Review by Kshitij1K — APPROVED** [review]
Cool, just a suggestion we should probably have empty files in other directories as well (and their corresponding sections in the README), just so that people know that they are supposed to exist and their purpose.

**Pull request merged**